Tchap/Modules/Application/AppDelegate.swift

```diff
@@ -109,7 +109,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return
         }
         
-        _ = self.appCoordinator.showRoom(with: roomId)
+        _ = self.appCoordinator.resumeBySelectingRoom(with: roomId)
```

Comment:
Remove usage of deprecated methods `application(_:handleActionWithIdentifier:for:completionHandler:)` and `application(_:didReceive:)` and use `userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:` to navigate to a room instead.

Reply:
done during the rebase (develop_rebase branch)